test.txt

```diff
@@ -1 +1,3 @@
-asdfasdfasdfasdf
\ No newline at end of file
+asdfasdfasdfasdfaasdfaasdfasdfasdf
+
+asdfaaa
```

Comment:
We already have enough of the left-hand characters. How about some lkhigujkhlk?
I can't approve this PR until this is resolved.